src/service/riskScoring.js

```diff
@@ -0,0 +1,39 @@
+import Fingerprint from 'fingerprintjs2';
+import request from '@/class/singleton/request';
+import { METRIC_TYPES } from '@/constants';
+
+/**
+ * @param {object[]} entries
+ * @returns {void}
+ */
+const sendFingerprint = () => {
+  const finger = new Fingerprint({
+    excludeCanvas: true,
+  });
+
+  finger.get(async (hash, entries) => {
+    /**
+     * @type {{
+     *   [key: string]: string|number,
+     * }}
+     */
+    const json = entries.reduce(
+      (accumulator, entry) => ({
+        [entry.key]: entry.value,
+        ...accumulator,
+      }),
+      {},
+    );
+
+    await request.post(`${ENV.VUE_APP_IDENTITY_API_URL}/user/metric`, [
+      {
+        payload: btoa(JSON.stringify(json)),
```

Comment:
if in json field will some unicode symbols, base64 can be not correct, better to use like this `window.btoa(unescape(encodeURIComponent(str)));`

see in [docs](https://developer.mozilla.org/ru/docs/Web/API/WindowBase64/btoa)

Reply:
yes, but we haven't unicode here, just numbers and latin letters

Reply:
done, but I guess, it's a similar to a foolproof a bit (extra layer of security even there aren't real hole)

Reply:
@kazurus should we add unicode support even it no unicode here? I guess it's overhead in this case

Reply:
yes, we will another metrics, and we don't want to observe it